app/src/main/res/values/strings.xml

```diff
@@ -347,4 +347,8 @@
     <string name="failed_saving_image">Failed to save image!</string>
     <string name="home_switch_anon">Switch to anonymous</string>
     <string name="login_already_logged_in">You are already logged in with this account</string>
+    <string name="person_profile_dm_person">Message this person</string>
```

Comment:
`Message <username>` is preferable

Reply:
It's because the other two options are X this person

Reply:
Message <username> can get possibly pretty long depending on the username, what about `Send message` ?